docs/UsersGuide/source/tables/hafs_cdeps_config.csv

```diff
@@ -0,0 +1,35 @@
+﻿Section,Variable name,Description
+[config],,
+,run_datm,Whether to run the data atmosphere.
+,run_docn,Whether to run the data ocean.
+,run_ocean,Whether to run the active ocean model. Must be no if run_docn=yes.
+,run_dwav,Whether to run the data wave. Not implemented.
+,make_mesh_atm,"Whether the workflow should generate a mesh file that describes the grid for the data atmosphere. Unless you are providing a custom mesh file, this should be set to yes. No effect if run_datm=no."
+,mesh_atm_in,The location of the premade datm mesh file. Only used if run_datm=yes and make_mesh_atm=no.
+,make_mesh_ocn,"Whether the workflow should generate a mesh file that describes the grid for the data ocean. Unless you are providing a custom mesh file, this should be set to yes. No effect if run_docn=no."
+,mesh_ocn_in,The location of the premade docn mesh file. Only used if run_docn=yes and make_mesh_ocn=no.
+,datm_source,The data source used for DATM. Only ERA5 is supported. No effect if run_datm=no.
+,DATMdir,The location where DATM input data are staged by the user. This variable is set in system.conf.[machine]. The workflow will not download new data if the necessary input files are already present in DATMdir.
+,docn_source,The data source used for DOCN. Only OISST and GHRSST are supported. No effect if run_docn=no.
+,DOCNdir,The location where DOCN input data are staged by the user. This variable is set in system.conf.[machine]. The workflow will not download new data if the necessary input files are already present in DOCNdir.
+,scrub_com,Whether to scrub the cycle’s com directory at the end of the run. Recommend setting to no to avoid losing files generated by CDEPS that the archive job does not save.
+,scrub_work,Whether to scrub the cycle’s work directory at the end of the run. Recommend setting to no to avoid losing files generated by CDEPS that the archive job does not save.
+,run_vortexinit,Whether to run the vortex initialization. Must be no if run_datm=yes.
+,run_gsi_vr,Whether to run the GSI-based vortex relocation. Must be no if run_datm=yes.
+,run_gsi_vr_fgat,Whether to run the GSI-based vortex relocation with FGAT. Must be no if run_datm=yes.
+,run_gsi_vr_ens,Whether to run the GSI-based vortex relocation for each HAFS ensemble member. Must be no if run_datm=yes.
+,run_gsi,Whether to run data assimilation with GSI. Must be no if run_datm=yes.
+,run_fgat,Whether to run data assimilation using FGAT. Must be no if run_datm=yes.

```

Comment:
What is FGAT?

Reply:
@gspetro-NOAA FGAT is First-Guess at the Appropriate Time. You can get some idea from [HWRF Users Guide](https://dtcenter.org/sites/default/files/community-code/hwrf/docs/scientific_documents/HWRFv4.0a_ScientificDoc.pdf) (see page 30). There may be some HAFS related documentation which I am unaware.